### DISCUSSION TITLE
Is it allowed to `send()` event with payload using `@xstate/fsm`?

### DISCUSSION THREAD

**Discussion by frederikhors** [statelyai/xstate#2517] [Q&A] [answered]
I'm using xstate for the first time today.

I'm using `@xstate/fsm`.

Is it allowed to `send('EVENT_NAME', {payload})`?

I'm getting errors doing this.

**Comment by davidkpiano** [comment, marked as answer]
It is not. You need to send it as an event object:

```js
send({ type: 'EVENT_NAME', payload });
```

**Reply by frederikhors** [reply]
It doesn't work. Hence my question: is it allowed? I cannot find examples in docs (for fsm).

**Reply by davidkpiano** [reply]
It is not allowed. You can only send an event object or a simple event type (no payload).

See this example: https://stackblitz.com/edit/js-qfxwwc?file=index.js

**Reply by frederikhors** [reply]
Ok. It works with: `service.send({ type: 'EVENT', value: {...payload} });`

Thanks!
